improver_tests/ensemble_copula_coupling/test_ConvertLocationAndScaleParametersToPercentiles.py

```diff
@@ -227,22 +227,22 @@ def test_simple_data_truncnorm_distribution(self):
         current_forecast_predictor.data = current_forecast_predictor.data + 1
         # Use an adjusted version of the ensemble variance as a proxy for the
         # scale parameter for the truncated normal distribution.
-        current_forecast_variance = self.temperature_cube.collapsed(
-            "realization", iris.analysis.VARIANCE
+        current_forecast_stddev = self.temperature_cube.collapsed(
```

Comment:
Do you think it's better to make this maths equivalent to what it was doing before (i.e. calculate variance +1 then square root) so the results of this test don't change?

Reply:
I've haven't made this particular change out of a concern that the resulting test might be a bit more complicated overall, but I can see how this might be helpful for the diff for this particular PR. In this file, though I have made a number of other docstring corrections.